src/Common/ErrorCodes.cpp

```diff
@@ -637,6 +637,7 @@
     M(666, CANNOT_USE_CACHE) \
     M(667, NOT_INITIALIZED) \
     M(668, INVALID_STATE) \
+    M(669, BACKGROUND_CACHE_WRITE_WAS_CANCELLED) \
```

Comment:
What do you think about using existing `UNFINISHED` code? We use it for cancelled background operations like merges and killed mutations. (there's also `QUERY_WAS_CANCELLED`, but it's more specific)

Reply:
> What do you think about using existing UNFINISHED code?

Seems good, will use it. 👍🏻